### DISCUSSION TITLE
go daddy shared hosting help

### DISCUSSION THREAD

**Discussion by Shrinathmukund** [snipe/snipe-it#15520] [General]
I am getting the following error when I run ./install.sh pls help
<img width="1512" alt="Screenshot 2024-09-18 at 12 22 06" src="https://github.com/user-attachments/assets/79725ba7-08d1-4ff2-85db-68d293180955">

**Comment by Shrinathmukund** [comment]
guys can someone help in installing the same thanks

**Reply by simisola88** [reply]
@Shrinathmukund  follow the steps here 
https://snipe-it.readme.io/docs/installation

ensure your domain name is pointing to the public folder
also confirm the PHP version your terminal is using is the right one  
"php -v" would get you the one it's using
